src/App.js

```diff
@@ -47,7 +50,7 @@ export default function App() {
           path="/"
           render={() => (
             isUserLoggedIn() ? (
-              <Home />
+              <Home isAd={isAdmin} />
```

Comment:
This is a minor thing, but could you please change all instances of isAd to isAdmin? It just makes reading/following the code easier.